templates/contrib/jdbc/Makefile

```diff
@@ -1,4 +1,4 @@
-BUILD_ENGINE ?= podman
+BUILD_ENGINE ?= docker
```

Comment:
I thought we were supposed to be promoting podman instead of docker, but I think this is fine if it's what works.

Reply:
cekit does not work very well with podman yet =/